zerver/openapi/python_examples.py

```diff
@@ -372,6 +372,29 @@ def add_realm_filter(client: Client) -> None:
     validate_against_openapi_schema(result, "/realm/filters", "post", "200")
 
 
+@openapi_test_function("/realm/filters/{filter_id}:patch")
+def update_realm_filter(client: Client) -> None:
+
+    # {code_example|start}
+    # Update the linkifier (realm_filter) with ID 1
+    filter_id = 1
+    request = {
+        "pattern": "#(?P<id>[0-9]+)",
+        "url_format_string": "https://github.com/zulip/zulip/issues/%(id)s",
+    }
+
+    result = client.call_endpoint(
+        url=f"/realm/filters/{filter_id}", method="PATCH", request=request
+    )
+    # {code_example|end}
+
+    validate_against_openapi_schema(result, "/realm/filters/{filter_id}", "patch", "200")
+
+    # Test `Filter not found` error
+    result = client.call_endpoint(url=f"/realm/filters/{filter_id}", method="PATCH")
+    validate_against_openapi_schema(result, "/realm/filters/{filter_id}", "patch", "400")
```

Comment:
I do not understand how using the same linkifier ID gives a `200` response at one time and a `400: Filter not found` at another time. Do we or do we not have a linkifier with `id=1` in the database?
I think we don't need to test bad inputs here, because the point of this file is to generate correct examples.
You can probably create a new linkifier outside the `code_example` block at the beginning of the test, and then use the ID you get there to update.

Reply:
> I do not understand how using the same linkifier ID gives a 200 response at one time and a `400: Filter not found` at another time.

That's because in the 2nd call to `client.call_endpoint` I'm not passing `request=request`. But I guess I confused here by not doing `{filter_id + 1}`. Sorry for the confusion.

Reply:
Ok, that makes sense. But I still think there's no need to test the `Filter not found` case, because it doesn't add to the documentation.

Reply:
Cool! I should better remove that part. Also, since `*_realm_filter` functions do not do the same.